File: python/laba_1/zadanie_1.py
print ('Введите переменные с клавиатуры')
x1,x2,x3,y1,y2,y3 = map(int,input().split())

a = (((x2 - x1)**2 +(y2 - y1)**2)**0.5)
print("Сторона a =", a)

b = (((x3 - x2)**2 +(y3 - y2)**2)**0.5)
print("Сторона b =", b)

c = (((x1 - x3)**2 +(y1 - y3)**2)**0.5)
print("Сторона c = ", c)

p = a+b+c
print("Пеpиметр = ",p)
pl = int(p/2) #Полупериметр

S = ((pl*(pl-a)*(pl-b)*(pl-c))**0.5)
print("Площадь = ",S)

File: Java/laba1/zadanie_1.java
//Дана строка. Напечатать в алфавитном порядке все слова из данной
//строки, имеющие заданную длину n.
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.io.StreamTokenizer;
import java.util.Arrays;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader breader = new BufferedReader(new InputStreamReader(System.in));
        int n = 5;
        StreamTokenizer st = new StreamTokenizer(breader);
        st.ordinaryChar('/');
        String a = "";
        String[] array = {};
        int count = 0;
        while(st.ttype != StreamTokenizer.TT_EOF){
            if (st.ttype == StreamTokenizer.TT_WORD){
                if (st.sval.length() == n){
                    a = a.concat(st.sval);
                    a = a.concat(" ");
                }
            }
            st.nextToken();
        }
        String[] out = a.split(" ");
        Arrays.sort(out);
        for (int i = 0;i<out.length;i++) {
            System.out.println(out[i]);
        }

    }


}
//Дана строка, содержащая текст. Найти слово, встречающееся в
//каждом предложении, или сообщить, что такого слова нет.
